api/bash/create-teams.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Replace the "xxxxx" with the required values
+# Author: @ppremk
+
+# Script to create GitHub Teams in bulk on GitHub.com Organization
+# PAT Tokens needs to have the correct scope to be able to create teams in an organization
+# Teams are added as an Array. Teams are created as stand alone teams. Team relationship is not defined
+
+# To run the script:
+#
+# - Update VARS section in script
+# - chmod +x script.sh
+# - ./script.sh
+
+# VARS
+ORGNAME="xxxx" 
+PATTOKEN="xxxx"
+TEAMS=("team-name-1" "team-name-2")
```

Comment:
minor nit: in bash script you want to use all lower case variables as upper case variables could be used by the system (not happening here... but it is a good practice I think)

Reply:
implemented as review fix